src/cancelContentPlay.js

```diff
@@ -37,6 +37,16 @@ const cancelContentPlay = function(player) {
       player.el_.style.display = 'block';
       placeholder.remove();
     });
+
+    // On iOS 10 on iPhone, playsinline is added when pinching to return from
+    // fullscreen. Detect fullscreen change, remove placeholder and show player.
+    player.on('fullscreenchange', function () {
+      if (player.el_.hasAttribute('playsinline') && placeholder) {
```

Comment:
Just noticed something. Can we also have a check over here: `!player.el_.hasAttribute('playsinline')` so that on fullscreen change black div is not seen even in the case if `playsinline` is not set (in case of iphone 8 and 9)? Does that sound ok?